kubectl panic: runtime error: invalid memory address or nil pointer dereference

This is on GKE. Everything at version 1.2.2.

```
$ kubectl rollout history deployment nginx
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0xb8 pc=0xe1affa]

goroutine 1 [running]:
k8s.io/kubernetes/pkg/util/deployment.Revision(0x0, 0xc2083a2420, 0x0, 0x0)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/util/deployment/deployment.go:328 +0x4a
k8s.io/kubernetes/pkg/kubectl.(*DeploymentHistoryViewer).History(0xc20812ccb0, 0x1623b40, 0x7, 0x7ffdd08fdee9, 0x5, 0x0, 0x0, 0x0)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/kubectl/history.go:80 +0x7ea
k8s.io/kubernetes/pkg/kubectl/cmd/rollout.RunHistory(0xc208134960, 0xc2081ada00, 0x7fd3570f4030, 0xc20804e008, 0xc2081af1c0, 0x2, 0x2, 0xc2081ae260, 0x0, 0x0)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/kubectl/cmd/rollout/rollout_history.go:98 +0x8a8
k8s.io/kubernetes/pkg/kubectl/cmd/rollout.func·002(0xc2081ada00, 0xc2081af1c0, 0x2, 0x2)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/pkg/kubectl/cmd/rollout/rollout_history.go:55 +0x6f
github.com/spf13/cobra.(*Command).execute(0xc2081ada00, 0xc2081af040, 0x2, 0x2, 0x0, 0x0)
        /go/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/spf13/cobra/command.go:572 +0x82f
github.com/spf13/cobra.(*Command).ExecuteC(0xc20806d600, 0xc2081ada00, 0x0, 0x0)
        /go/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/spf13/cobra/command.go:662 +0x4db
github.com/spf13/cobra.(*Command).Execute(0xc20806d600, 0x0, 0x0)
        /go/src/k8s.io/kubernetes/Godeps/_workspace/src/github.com/spf13/cobra/command.go:618 +0x3a
k8s.io/kubernetes/cmd/kubectl/app.Run(0x0, 0x0)
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kubectl/app/kubectl.go:28 +0x101
main.main()
        /go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/cmd/kubectl/kubectl.go:28 +0x2f
```

```
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"2", GitVersion:"v1.2.2", GitCommit:"528f879e7d3790ea4287687ef0ab3f2a01cc2718", GitTreeState:"clean"}
Server Version: version.Info{Major:"1", Minor:"2", GitVersion:"v1.2.2", GitCommit:"528f879e7d3790ea4287687ef0ab3f2a01cc2718", GitTreeState:"clean"}
```

@ivanpi-mtv @kenashcraft

--- a/pkg/kubectl/history.go
+++ b/pkg/kubectl/history.go
@@ -76,7 +76,10 @@ func (h *DeploymentHistoryViewer) History(namespace, name string) (HistoryInfo, 
 	if err != nil {
 		return historyInfo, fmt.Errorf("failed to retrieve new replica set from deployment %s: %v", name, err)
 	}
-	allRSs := append(allOldRSs, newRS)
+	allRSs := allOldRSs
+	if newRS != nil {
+		allRSs = append(allRSs, newRS)
+	}
 	for _, rs := range allRSs {
 		v, err := deploymentutil.Revision(rs)
 		if err != nil {

